internal/schemautil/service.go

```diff
@@ -693,11 +693,12 @@ func FlattenServiceComponents(r *aiven.Service) []map[string]interface{} {
 
 	for _, c := range r.Components {
 		component := map[string]interface{}{
-			"component": c.Component,
-			"host":      c.Host,
-			"port":      c.Port,
-			"route":     c.Route,
-			"usage":     c.Usage,
+			"component":                   c.Component,
+			"host":                        c.Host,
+			"port":                        c.Port,
+			"route":                       c.Route,
+			"usage":                       c.Usage,
+			"kafka_authentication_method": c.KafkaAuthenticationMethod,
```

Comment:
could this please be moved to be in a specific service that this is related to?

Reply:
never mind, this would require serious refactoring, and is inappropriate to be done in v3.x, since we already have the field in every service